CONTRIBUTING.rst

```diff
@@ -199,7 +199,7 @@ Short version
 #. Fork the repository.
 #. Fetch tags from upstream if necessary (if you cloned only main `git fetch --tags https://github.com/pytest-dev/pytest`).
 #. Enable and install `pre-commit <https://pre-commit.com>`_ to ensure style-guides and code checks are followed.
-#. Follow **PEP-8** for naming and `black <https://github.com/psf/black>`_ for formatting.
+#. Follow `PEP-8 <https://www.python.org/dev/peps/pep-0008/>`_ for naming and `black <https://github.com/psf/black>`_ for formatting.
```

Comment:
We can leave out the mention to black, as we will autoformat as part of the PR process.

```suggestion
#. Follow `PEP-8 <https://www.python.org/dev/peps/pep-0008/>`_ for naming.
```